src/Components/__tests__/ArtsyShippingEstimate.jest.tsx

```diff
@@ -44,24 +47,24 @@ jest.mock("Utils/Hooks/useLoadScript", () => {
 })
 
 const validArtworkData = {
-  depthCm: null,
-  diameterCm: null,
-  framedHeight: null,
-  framedWidth: null,
-  framedDepth: null,
-  framedDiameter: null,
+  depthCm: undefined,
```

Comment:
Is this due to the API regarding null as a value, vs undefined being stripped somehow?